docs/user-guides/configuration.md

```diff
@@ -223,24 +250,25 @@ gateway:
 
 #### Index replica
 
-`gateway.lb.gateway_config.index_replica` represents how many agent pods that a vector will be inserted into.
+`gateway.lb.gateway_config.index_replica` represents how many Vald Agent pods a vector will be inserted into.
+The maximum value of the index replica should be one-third of the Vald Agent pods deployed.
 
 ```yaml
 gateway:
   lb:
     gateway_config:
-      index_replica: 3
+      index_replica: 3 // By setting the index replica to 3, the number of Vald Agent pods deployed should be more than 9.
```

Comment:
I think it would be better to describe value as a percentage.
in this case it would be 30%